src/markdown/tutorial/part-1/01-orientation.md

```diff
@@ -46,12 +46,12 @@ Hack: make an empty package.json to convince ember-cli we are really not in an E
 # pretend we are running NPM.
 
 #[cfg(all(ci, unix))]
-#[display(ember new super-rentals --lang en)]
-ember new super-rentals --lang en --pnpm \
+#[display(ember new super-rentals --lang en --strict)]
```

Comment:
This will be default in the vite blueprint, but it's good to have for now (as the vite blueprint is still an experiment in ember-cli)